src/com/mambu/apisdk/util/ParamsMap.java

```diff
@@ -63,7 +60,7 @@ public String getURLString() {
 
 				try {
 					// URL encode values
-					encodedValue = URLEncoder.encode(value, HTTP.UTF_8);
+					encodedValue = URLEncoder.encode(value, StandardCharsets.UTF_8.name());
 

```

Comment:
Good idea to use the StandardCharsets!  May be you can then also change this in RequestExecutorImpl (UTF8_charset definition)?